juno/assetlist.json

```diff
@@ -206,7 +206,29 @@
         "symbol": "ASVT",
         "logo_URIs": {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/asvt.png"
-        }
+        },
+           {
+        "description": "DAO dedicated to building tools on the Juno Network",
+        "type_asset": "cw20",
+        "address": "juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3",
+        "denom_units": [
+          {
+            "denom": "cw20:juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3",
+            "exponent": 0
+          },
+          {
+            "denom": "joe",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3",
+        "name": "Joe",
```

Comment:
sorry, I wasn't clear. The name should remain as "JoeDAO", as is there in the contract. It was just that the display property had to be one of the defined denom_units, which is why I mentioned that it should be "joe". So although display is correct, the name should be back to "JoeDAO"